Sources/Paywalls/Components/PaywallComponentBases.swift

```diff
@@ -125,16 +96,6 @@ public enum PaywallComponent: Decodable, Sendable, Hashable, Equatable {
             return component.focusIdentifiers
         case .image(let component):
             return component.focusIdentifiers
-        case .video(let component):
-            return component.focusIdentifiers
```

Comment:
Same with this! This was for game controller POC so this whole `focusIdentifier` thing can get deleted 👋

Reply:
Ya I was thinking that, will do.